OpenRA.Mods.Common/Widgets/Logic/Ingame/Hotkeys/PauseHotkeyLogic.cs

```diff
@@ -30,8 +30,8 @@ public PauseHotkeyLogic(Widget widget, ModData modData, World world, Dictionary<
 
 		protected override bool OnHotkeyActivated(KeyInput e)
 		{
-			// Disable pausing for spectators and defeated players unless they are the game host
-			if (!Game.IsHost && (world.LocalPlayer == null || world.LocalPlayer.WinState == WinState.Lost))
+			// Disable pausing for spectators and players unless they are the game host.
```

Comment:
Please maintain the original comment and the behaviour it refers to.  Changing this isn't necessary as part of fixing the bug.